tests/xtradbcluster/xtra_db_cluster_test.go

```diff
@@ -62,17 +63,17 @@ func TestXtraDBClusterAPI(t *testing.T) {
 		},
 		Name: name,
 		Params: &controllerv1beta1.XtraDBClusterParams{
-			ClusterSize: 1,
```

Comment:
is this changes required?

Reply:
Set back to 2

Reply:
I didn't get you, what do you mean?
what about CpuM?

Reply:
I've returned the clustersize to 1 and cpuM to 200 as the original test.